calico-policies/private-network-isolation/calico-v3/us-east/allow-egress-pods-private.yaml

```diff
@@ -95,6 +95,7 @@ spec:
       - 166.9.22.55/32
       - 166.9.22.8/32
       - 166.9.22.9/32
+      - 166.9.22.216/32
```

Comment:
These are still the wrong IPs, see https://github.com/IBM-Cloud/kube-samples/pull/190 for how to find the right ones